etc/profile-m-z/rtv-addons.inc

```diff
@@ -0,0 +1,23 @@
+# This file is overwritten during software install.
+# Persistent customizations should go in a .local file.
+
+# You can configure rtv to open different type of links
+# in external applications. Configuration here:
+# https://github.com/michael-lazar/rtv#viewing-media-links
+# This include is meant to facilitate that configuration
+# with the use of a .local file.
+
+ignore nosound
+ignore private-bin
+ignore dbus-system none
```

Comment:
is granting access to the system bus really needed?

Reply:
lol it's not